sdk/digitaltwins/ci.yml

```diff
@@ -0,0 +1,30 @@
+# DO NOT EDIT THIS FILE
+# This file is generated automatically and any changes will be lost.
+trigger:
+  branches:
+    include:
+      - master
```

Comment:
You'll probably want to these triggers as well.

```
- feature/*
- hotfix/*
- release/*
```

This will ensure that feature branches, release, and hotfix branches will run CI as well. This matches our goal of how to handle branching management as well.